Readme.md

```diff
@@ -1,33 +1,39 @@
-<p>All code related to selection resides in the <strong>GridCheckMarksSelection </strong>helper class. So, to implement this functionality in your application, you can simply add this class to your project and then create a new <strong>GridCheckMarksSelection </strong>instance.<br>Please note if you use the <strong><a href="http://documentation.devexpress.com/#WindowsForms/DevExpressXtraGridViewsGridGridViewAppearances_EvenRowtopic"><u>AppearanceEvenRow</u></a> </strong>and <strong><a href="http://documentation.devexpress.com/#WindowsForms/DevExpressXtraGridViewsGridGridViewAppearances_OddRowtopic"><u>AppearanceOddRow</u></a> </strong>styles, by default they have a higher priority than the <strong><a href="http://documentation.devexpress.com/#WindowsForms/DevExpressXtraGridViewsGridGridView_RowStyletopic"><u>RowStyle</u></a> </strong>event used for the selection in this example. To avoid drawing incorrect selected rows, enable the <strong><a href="http://documentation.devexpress.com/#WindowsForms/DevExpressXtraGridViewsGridRowStyleEventArgs_HighPrioritytopic"><u>e.HighPriority</u></a></strong> option. <br><br>The <strong>GridCheckMarksSelection </strong>class also provides different ~<strong>Select</strong> methods which allow selecting data and group rows in code. To obtain selected rows, you can use the <strong>SelectedCount</strong> property and <strong>GetSelectedRow</strong> method.</p>
-
-<br/>
-
-=======
 <!-- default badges list -->
 ![](https://img.shields.io/endpoint?url=https://codecentral.devexpress.com/api/v1/VersionRange/128632513/18.1.3%2B)
 [![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/E1271)
 [![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
 <!-- default badges end -->
 
+# Data Grid for Windows Forms - Multiple row selection using checkboxes (web style row selection)
+
+## Built-in Checkbox Column
+In version 13.2 and higher, Data Grid supports a built-in checkbox column for multiple row selection in grid Views. To activate this checkbox column, enable the [GridView.OptionsSelection.MultiSelect](https://documentation.devexpress.com/WindowsForms/DevExpress.XtraGrid.Views.Base.ColumnViewOptionsSelection.MultiSelect.property) option, and set the [GridView.OptionsSelection.MultiSelectMode](https://documentation.devexpress.com/WindowsForms/DevExpress.XtraGrid.Views.Grid.GridOptionsSelection.MultiSelectMode.property) property to `CheckBoxRowSelect`. 
+
+Starting with version 18.1, you can sync row selected states with a Boolean field in the bound data source. Checkbox column bound mode ensures that row selection persists whenever you filter or group grid data.
```

Comment:
I think we're supposed to avoid the "Starting with vXX.x" pattern. How about the following?<br>
Also, can we change "Checkbox column bound mode" to "A checkbox column's bound mode"?

`In v18.1 and later versions, you can sync row selected states with a Boolean field in the bound data source. A checkbox column's bound mode ensures that row selection persists whenever you filter or group grid data.`

Reply:
```suggestion
In v18.1 and later versions, you can sync row selected states with a Boolean field in the bound data source. A checkbox column's bound mode ensures that row selection persists whenever you filter or group grid data.
```